strings/french.json

```diff
@@ -420,8 +423,8 @@
   "*This value was automatically adjusted because your group is too small for the disagreement number.": "*Cette valeur a été ajustée automatiquement car la taille de votre groupe est trop petite pour le seuil de désaccord.",
   "Active": "Actif",
   "Change to this system": "Choisir ce système",
-  "Maximum number of \"no\" votes:": "Nombre maximal de votes négatifs :",
-  "Percentage of members that need to agree:": "Pourcentage de votes positifs requis :",
+  "Maximum number of \"no\" votes:": "Nombre maximal de votes négatifs:",
+  "Percentage of members that need to agree:": "Pourcentage de votes positifs requis:",
```

Comment:
I think we decided that the spaces should exist for French, right @pieer @snowteamer ?